build/testing/applitools/run-msft-docs.js

```diff
@@ -7,9 +7,9 @@ let spawn = require("child_process").spawnSync;
  */
 class Config {
     constructor() {
-        this.domainName = "https://msft-docs.fast-dna.net";
-        this.appName = "FAST-DNA";
-        this.siteName = "MSFT Doc Site";
+        this.domainName = "https://explore.fastdna.net";
+        this.appName = "FASTDNA";

```

Comment:
Can you give me a rundown on changing the hyphenation here?

Reply:
In most new resources (domains, sites, etc) we're going with fastdna instead of the use of hyphens.  In the Name i'm not sure this is true. But, absolutely in pathing per @Falkicon.

Reply:
I get the pathing - that makes sense. Our name everywhere is hyphenated, so I feel like we should probably confirm that *is not* changing ASAP so we can be consistent with branding.